### ISSUE TITLE
Uh. Can we move this to Glitch or GitLab or something?

### ISSUE THREAD

**Issue by EvanBurchard** [hankchizljaw/companies-that-work-with-ice#13]
Seems like GitHub is on the list of companies now, so might be worth exploring another option?

**Comment by hankchizljaw** [comment]
Yeh for sure. Incredibly frustrating, because GitHub is a very accessible way of running this sort of thing.

Thought I’d being put into it.

**Comment by EvanBurchard** [comment]
Woah. scratch gitlab holy shit.

**Comment by jalcine** [comment]
It could be definitely done on Glitch. There's a way to have people "join" a project and if it's something that can be built locally to a static site, it can be even faster.

**Comment by hankchizljaw** [comment]
Can you do PRs etc with glitch? A handy thing that this platform gives is plenty of room for discussion, so we don't inadvertently mislead people.

**Comment by jalcine** [comment]
Not really, but you can see the history of a project and who did what when you view the rewind history of apps.

**Comment by hankchizljaw** [comment]
My only worry with this is that if I don't get around to checking changes, live info could be incorrect for a few days. I love Glitch, but I'm not 100% convinced it'll be ideal for this project, annoyingly.

**Comment by ultraloveninja** [comment]
What about BitBucket or Gitea?

https://gitea.io/en-us/
